src/js/hmrc/mtdFraudPrevention.js

```diff
@@ -80,6 +82,10 @@ export const getFraudPreventionHeaders = async () => {
       header: fraudPreventionHeadersEnum.DEVICE_LOCAL_IPS,
       callback: async () => encodeURI(await getDeviceLocalIPAsString()),
     },
+    { 
+      header: fraudPreventionHeadersEnum.DEVICE_LOCAL_IPS_TIMESTAMP,
+      callback: async () => encodeURI(await getDeviceLocalIPTimeStamp()),
```

Comment:
This is the correct API but we'll need to do something clever to store the value of the IP and its timestamp so regardless of which order these get called it we only collect the IP's once (it can take a bit of time)

Reply:
Also, we don't need to encode the timestamp, am I right?